### PR TITLE
Check organization owner's organization in backend when creating

### PR DESCRIPTION
**Pull request by okahilak**
Part of #1893 

Related to, but independent of, #1925.

# Definition of Done / Review checklist

## Reviewability
- [x] link to issue
- [x] note if PR is on top of other PR

## Documentation
- [ ] update changelog if necessary
   - will do after all organization changes are done

## Testing
- [x] complex logic is unit tested
- [x] valuable features are integration / browser / acceptance tested automatically

### CODE REVIEW

File: src/clj/rems/api/services/util.clj [hunk 1]
```diff
@@ -0,0 +1,46 @@
+(ns rems.api.services.util
+  (:require [clojure.test :refer [deftest is testing]]
+            [rems.db.roles :as roles]
+            [rems.db.users :as users]))
+
+(defn forbidden-organization? [user-id organization]
+  (let [user (users/get-user user-id)]
+    (when (and (contains? (roles/get-roles user-id) :organization-owner)
```
Comment (opqdonut) [inline src/clj/rems/api/services/util.clj]: hmmm I think we need to allow owners to create anything, so perhaps: `(and (not (contains? roles :owner)) ...)`

File: src/clj/rems/api/services/util.clj [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+                                 "organization-owner" #{:organization-owner}
+                                 "organization-owner-with-no-organization" #{:organization-owner}}
+                users/get-user {"owner" {:eppn "owner"
+                                         :organization "own organization"}
+                                "organization-owner" {:eppn "organization-owner"
+                                                      :organization "own organization"}
+                                "organization-owner-with-no-organization" {:eppn "organization-owner-with-no-organization"}}]
+    (let [forbidden-error {:success false
+                           :errors [{:type :t.administration.errors/forbidden-organization}]}]
+      (testing "for owner, all organizations are permitted"
```
Comment (opqdonut) [inline src/clj/rems/api/services/util.clj]: should also test multiple roles, e.g. owner and organization-owner

File: src/clj/rems/api/services/util.clj (hunk 1 above)
Comment (okahilak) [reply src/clj/rems/api/services/util.clj]: Good point, done.

File: src/clj/rems/api/services/util.clj (hunk 2 above)
Comment (okahilak) [reply src/clj/rems/api/services/util.clj]: Done.

**Pull request merged**